internal/pkg/api/server/buffer/dependencies/store.go

```diff
@@ -65,3 +72,62 @@ func (c *ConfigStore) CreateReceiver(ctx context.Context, receiver model.Receive
 
 	return nil
 }
+
+// GetReceiver fetches a receiver from the store.
+//
+// This method returns nil if no receiver was found.
+func (c *ConfigStore) GetReceiver(ctx context.Context, projectId int, receiverId string) (r *model.Receiver, err error) {
+	logger, tracer, client := c.logger, c.tracer, c.etcdClient
+
+	_, span := tracer.Start(ctx, "kac.api.server.buffer.dependencies.store.GetReceiver")
+	defer telemetryUtils.EndSpan(span, &err)
+
+	key := ReceiverKey(projectId, receiverId)
+
+	logger.Debugf(`GET "%s"`, key)
+	resp, err := client.KV.Get(ctx, key)
+	if err != nil {
+		return nil, err
+	}
+
+	// No receiver found
+	if len(resp.Kvs) == 0 {
+		logger.Debugf(`No receiver "%s" found`, key)
+		return nil, nil
+	}
+
+	logger.Debugf(`Decoding "%s"`, key)
+	receiver := &model.Receiver{}
+	if err = json.DecodeString(string(resp.Kvs[0].Value), receiver); err != nil {
+		return nil, err
+	}
+
+	return receiver, nil
+}
+
+func (c *ConfigStore) ListReceivers(ctx context.Context, projectId int) (r []*model.Receiver, err error) {
+	logger, tracer, client := c.logger, c.tracer, c.etcdClient
+
+	_, span := tracer.Start(ctx, "kac.api.server.buffer.dependencies.store.ListReceivers")
+	defer telemetryUtils.EndSpan(span, &err)
+
+	key := ProjectKey(projectId)
```

Comment:
Máme konvenci (zřejmě ji má i Go nebo je to aspoň obecná shoda) že akronymy a ostatní zkratky v názvech proměnných zůstávají velkými písmeny, tj. mělo by být `projectID`.

Reply:
Upravím, ale můžeme tohle řešit přes lintery? Podobný stylový diskuze máme docela často

Reply:
🤔 Tak to nevím jak moc dobře by zrovna tohle šlo nastavit v linteru. Jinak pokud vím, tak se držíme https://betterprogramming.pub/naming-conventions-in-go-short-but-descriptive-1fa7c6d2f32a (Pokud jsme ti to nedávali ke čtení, tak jsme to měli udělat. 🙂 )

Reply:
> Trochu "blbý" search mi našel, že máme dokonce víc _Id než _ID

Ano :slightly_smiling_face: na niektore standardny sme prisli az neskor a postupne ich opravujeme aj v starom kode.

Linter to podporoval/podporuje? https://github.com/golang/lint/issues/124

Neviem, @jprochazk mohol by si ten linter prosim doriesit/nastavit?

Reply:
> Linter to podporoval/podporuje? https://github.com/golang/lint/issues/124

Hledám, ale nic jsem nenašel.

Reply:
Vypadá to, že to řídí `stylecheck`, který pokud čtu správně nemáme zapnutý 
https://github.com/keboola/keboola-as-code/blob/ff9db26d1318a221171387d3cd5525f8a0373a11/build/ci/golangci.yml#L65-L71

https://golangci-lint.run/usage/linters/#stylecheck
https://staticcheck.io/docs/configuration/options/#initialisms

Reply:
> Trochu "blbý" search mi našel, že máme dokonce víc _Id než _ID

Opravujeme to postupně když jdeme do toho kódu něco řešit. Není to ideální, ale optimálnější než jít dělat masivní find & replace jen kvůli tomu. Tý jo, tak to by mě nenapadlo, že jsou lintery i na takovýhle věci. To v něm mají vyjmenovaný nejpoužívanější zkratky, nebo co? 😅 Ale tak tím líp.

(Kua, tady ty github komentáře jsou naprd, než napíšu odpověď tak mezi tím přibudou dvě od vás a všimnu si jich až kdo ví kdy v emailu. 🙈)

Reply:
> To v něm mají vyjmenovaný nejpoužívanější zkratky, nebo co?

Přesně tak, je tam seznam těch initialisms (např. HTTP, ID, URL). Já doufám, že to nebude find-and-replace ale jen `make fix`, protože to bude hlídat linter.

Reply:
Vytvořil jsem na to task https://keboola.atlassian.net/browse/KAC-286